src/OpenSage.Game/Data/Wav/DviApcmParser.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.IO;
+using OpenSage.Mathematics;
+
+namespace OpenSage.Data.Wav
+{
+    internal class DviAdpcmParser : WavParser
+    {
+        private int ByteToInt16(byte[] packed)
```

Comment:
If it actually takes **two bytes** and returns an **int32**, then `ByteToInt16` isn't really accurate. 🤔 Maybe call it something like `Int16FromLittleEndianBytes` and actually cast it to int16?